src/main/resources/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerStep/help.html

```diff
@@ -1,3 +1,8 @@
 <div>
-    Triggers a new build for a given job.
+    <p>
+        Triggers a new build for a given job.
```

Comment:
Which is the best form here? I would suggest infinitive, as next paragraph.

```suggestion
        Trigger a new build for a given job.
```

Reply:
Thanks for the review @MRamonLeon